js-packages/@prestojs/doc/package.json

```diff
@@ -19,8 +19,8 @@
     "clean": "rm -rf ./build && rm -rf tsconfig.tsbuildinfo"
   },
   "peerDependencies": {
-    "react": "*",
-    "react-dom": "*",
+    "react": ">= 16.12.0 < 19",
```

Comment:
Possibly just bump version and highlight breaking change React 18+?

Reply:
I mainly wanted to avoid this so existing projects we have can upgrade presto for any bugfixes without also having to upgrade React18 at same time. This should work fine with the caveat that presto will no longer avoid the 'setState after unmount' warnings for you